docs/topics/collection-aggregate.md

```diff
@@ -139,3 +139,21 @@ All reduce operations throw an exception on empty collections. To receive `null`
 * [`reduceRightOrNull()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/reduce-right-or-null.html)
 * [`reduceIndexedOrNull()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/reduce-indexed-or-null.html)
 * [`reduceRightIndexedOrNull()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/reduce-right-indexed-or-null.html)
+
+For cases where you want to save intermediate accumulator values, there are functions
+[`runningFold()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/running-fold.html) (or its synonym [`scan()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/scan.html)) 
+and [`runningReduce()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/running-reduce.html).
+
+```kotlin
+
+fun main() {
+    //sampleStart
+    val numbers = listOf(0, 1, 2, 3, 4, 5)
+    val runningReduceSum = numbers.runningReduce { sum, item -> sum + item }
+    val runningFoldSum = numbers.runningFold(10) { sum, item -> sum + item }
+    //sampleEnd
+    println(runningReduceSum.toString())
```

Comment:
The example could show better how we use intermediate values. For example, change the output so that it explains the result. Something like:
```
Sum of first N elements:
N = 1: 0
N = 2: 1
....
```
And similar output with an explanation for `runningFold()`

Reply:
Fixed.